src/map/entities/charentity.h

```diff
@@ -363,6 +365,7 @@ class CCharEntity : public CBattleEntity
 
     bool			m_isWeaponSkillKill;
     bool            m_isStyleLocked;
+    bool            m_isBlockingAid;
```

Comment:
where is the initialization? Either in the constructor (like everything else) or in here is fine